src/atomizer.js

```diff
@@ -295,37 +293,52 @@ Atomizer.prototype.parseConfig = function (config/*:AtomizerConfig*/, options/*:
                     value = `var(${matchVal.groups.cssVariable})`;
                 }
                 if (matchVal.groups.named) {
-                    // first check if 'inh' is the value
-                    if (matchVal.groups.named === 'inh') {
-                        value = 'inherit';
-                    }
                     // check if the named value matches any of the values
                     // registered in arguments.
-                    else if (rule.arguments && index < rule.arguments.length && Object.keys(rule.arguments[index]).indexOf(matchVal.groups.named) >= 0) {
+                    if (rule.arguments && index < rule.arguments.length && Object.keys(rule.arguments[index]).indexOf(matchVal.groups.named) >= 0) {
                         value = rule.arguments[index][matchVal.groups.named];
                     }
                     // now check if named value was passed in the config
                     else {
-                        propAndValue = [match.groups.atomicSelector, '(', matchVal.groups.named, ')'].join('');
+                        const propAndValue = [match.groups.atomicSelector, '(', matchVal.groups.named, ')'].join('');
                         let name;
 
-                        // no custom, warn it
-                        if (!config.custom) {
-                            warnings.push(propAndValue);
-                        }
-                        // as prop + value
-                        else if (Object.prototype.hasOwnProperty.call(config.custom, propAndValue)) {
-                            name = propAndValue;
+                        if (config.custom) {
+                            // as prop + value
+                            if (Object.prototype.hasOwnProperty.call(config.custom, propAndValue)) {
+                                name = propAndValue;
+                            }
+                            // as value
+                            else if (Object.prototype.hasOwnProperty.call(config.custom, matchVal.groups.named)) {
+                                name = matchVal.groups.named;
+                            }
                         }
-                        // as value
-                        else if (Object.prototype.hasOwnProperty.call(config.custom, matchVal.groups.named)) {
-                            name = matchVal.groups.named;
+                        value = utils.getCustomValue(config, name);
+
+                        if (!value) {
+                            // use global values if no custom value was found
+                            switch (matchVal.groups.named) {
```

Comment:
This might be more clear as a map instead of a switch

Reply:
a map?